x-pack/plugins/search_playground/public/components/message_list/assistant_message.tsx

```diff
@@ -53,53 +53,78 @@ export const AssistantMessage: React.FC<AssistantMessageProps> = ({ message }) =
   return (
     <>
       {!!retrievalDocs?.length && (
-        <EuiComment
-          username={username}
-          timelineAvatar="dot"
-          data-test-subj="retrieval-docs-comment"
-          eventColor="subdued"
-          css={{
-            '.euiAvatar': { backgroundColor: euiTheme.colors.ghost },
-            '.euiCommentEvent': {
-              border: euiTheme.border.thin,
-              borderRadius: euiTheme.border.radius.medium,
-            },
-          }}
-          event={
-            <>
+        <>
+          <EuiComment
+            username={username}
+            timelineAvatar="dot"
+            data-test-subj="assistant-message-searching"
+            eventColor="subdued"
+            css={{
+              '.euiAvatar': { backgroundColor: euiTheme.colors.ghost },
+              '.euiCommentEvent': {
+                border: euiTheme.border.thin,
+                borderRadius: euiTheme.border.radius.medium,
+              },
+            }}
+            event={
               <EuiText size="s">
                 <p>
                   <FormattedMessage
-                    id="xpack.searchPlayground.chat.message.assistant.retrievalDocs"
-                    defaultMessage="Grounding answer based on"
+                    id="xpack.searchPlayground.chat.message.assistant.searchingQuestion"
+                    defaultMessage='Searching "{question}"'
```

Comment:
@leemthompo what do you think about this message? Should this be something like "Searching for {question} in data sources" instead?

Reply:
this is what it looks like currently 
![Screenshot 2024-08-05 at 15 05 42](https://github.com/user-attachments/assets/b5955582-2c67-43be-bf1c-8b69859d3487)

Reply:
@joemcelroy I could see where this just says "Searching..." without printing the rewritten query at all, might be cleaner

Reply:
otherwise, yeh "Searching for" makes sense. Don't know if you need "in data sources" as sentence will get pretty long for longer queries

Reply:
i think the reason why we want to provide visibility of the query thats actually been used to search for. The LLM here has transformed the conversation into a query so useful to know.

Reply:
maybe could be more explicit in that case and have two separate bullets: 

- Transformed query: {query} (could _italicize_ the query to stand out)
- "Searching your data sources"

Reply:
added in https://github.com/elastic/kibana/pull/189848/commits/6a35192889611718e9edceffe4dde1f47359bc57

Reply:
@joemcelroy might be nice to have two bullets like I mentioned to first establish the transformed query